roles/usb_autorunner/files/autorunner.sh.j2

```diff
@@ -1,45 +1,46 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-AUTORUN_CONTENT_PATH={{ usb_autorunner_install_path }}/workdir
+TEMP_PATH=/tmp/autorunner/
+AUTORUNNER_WORKDIR={{ usb_autorunner_install_path }}/workdir
 PRIVATE_CERT={{ usb_autorunner_install_path }}/certificate/autorunner.pem
 PUBLIC_CERT={{ usb_autorunner_install_path }}/certificate/autorunner.pub.pem
 
-AUTORUN_FILENAME=startup.zip.enc
-KEY_FILENAME=secret.key.enc
+AUTORUN_FILENAME=autorun.zip.enc
+SECRET_KEY_FILENAME=secret.key.enc
 
-USB_DATA_PATH=$1
+MOUNT_POINT=$1
 
-if [ ! -r $USB_DATA_PATH/$KEY_FILENAME ]
+if [ ! -r $MOUNT_POINT/$SECRET_KEY_FILENAME ]
 then
-  echo "key file is missing, exiting"
+  echo "Secret key file '$SECRET_KEY_FILENAME' is missing at path $MOUNT_POINT/"
   exit 1
 fi
 
-if [ -r $USB_DATA_PATH/$AUTORUN_FILENAME ]
+if [ -r $MOUNT_POINT/$AUTORUN_FILENAME ]
 then
   
-  autorun_file_path=$(find $USB_DATA_PATH -maxdepth 1 -name $AUTORUN_FILENAME)
-  decrypt_key_path=$(find $USB_DATA_PATH -maxdepth 1 -name $KEY_FILENAME)
+  autorun_file_path=$(find $MOUNT_POINT -maxdepth 1 -name $AUTORUN_FILENAME)
+  decrypt_key_path=$(find $MOUNT_POINT -maxdepth 1 -name $SECRET_KEY_FILENAME)
   
   # Clean workdir
-  rm -f $AUTORUN_CONTENT_PATH/*
+  rm -f $AUTORUNNER_WORKDIR/*
+  rm -rf $TEMP_PATH
+  mkdir -p $TEMP_PATH
+  
   # Decrypting and executing the autorun script
+  openssl rsautl -decrypt -oaep -inkey $PRIVATE_CERT -in $decrypt_key_path -out $AUTORUNNER_WORKDIR/secret.key
+  openssl enc -d -aes-256-cbc -pbkdf2 -in $autorun_file_path -out $AUTORUNNER_WORKDIR/autorun.zip -pass file:$AUTORUNNER_WORKDIR/secret.key
```

Comment:
Could you change this to:
```
openssl enc -d -aes-256-cbc -md sha256 -in $autorun_file_path -out $AUTORUNNER_WORKDIR/autorun.zip -pass file:$AUTORUNNER_WORKDIR/secret.key
```

Removing `-pbkdf2` as this is not supported in OpenSSL on MacOS :/
Adding `-md sha256`to ensure compatiblity between OpenSSL versions